app/src/tests/components/contents/manageAccess/updatePermissions/AddPermissionForUpdateForm.test.tsx

```diff
@@ -67,33 +66,34 @@ describe("AddPermissionForUpdateForm", () => {
     });
   });
 
-  it("should submit form with correct values for packet.read permission and tag scope", async () => {
```

Comment:
the diff looks weird... its just commenting out 1 test and updating the other test to reference a packet rather than a tag